release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -845,6 +845,29 @@ All deprecated upstream Kubernetes metrics have been removed. The complete list
 
 High granularity request duration buckets were dropped in Prometheus, which were tracked with the `apiserver_request_duration_seconds_bucket` metric. This leaves enough buckets for meaningful alerting from other monitoring components, but drastically reduces cardinality.
 
+[id="ocp-4-4-removed-images-from-samples-imagestreams"]
+==== Removed images from samples imagestreams
+
+The following images are no longer included in the samples imagestreams provided with {product-title}:
+
+----
+registry.redhat.io/rhscl/nginx-110-rhel7
```

Comment:
We don't remove imagestream for `registry.redhat.io/rhscl/nginx-110-rhel7:latest` in ocp  4.4 version

Reply:
good catch @xiuwang 

@rolfedh we only removed registry.redhat.io/rhscl/nginx-112-rhel7:latest in 4.4 from 4.3 ... they had this odd thing where 1.10 was still supported but 1.12 was not

Reply:
@gabemontero  Okay, I cut these two from the list:
- registry.redhat.io/rhscl/nginx-110-rhel7
- registry.redhat.io/rhscl/nginx-112-rhel7
Good?

Reply:
No, remove registry.redhat.io/rhscl/nginx-110-rhel7 from the list, because it stayed in 4.4

Leave registry.redhat.io/rhscl/nginx-112-rhel7 in the list, because it was removed from 4.4

they had this odd situation where an older version remained, but a later version was removed

Reply:
> No, remove registry.redhat.io/rhscl/nginx-110-rhel7 from the list, because it stayed in 4.4
> 
> Leave registry.redhat.io/rhscl/nginx-112-rhel7 in the list, because it was removed from 4.4
> 
> they had this odd situation where an older version remained, but a later version was removed
Thanks, @gabemontero. Updated.